foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-KeyVaultPolicySetDefinition.json

```diff
@@ -214,9 +214,45 @@
                     "hsmLogCategories": {
                         "type": "string",
                         "defaultValue": "allLogs"
+                    },
+                    "keyVaultCertificateLifeTimeAction": {
+                        "type": "string",
+                        "defaultValue": "Disabled"
+                    },
+                    "keyVaultCertificateMaximumPercentageLife": {
+                        "type": "integer",
+                        "defaultValue": 80
+                    },
+                    "keyVaultCertificateMinimumDaysBeforeExpiry": {
+                        "type": "integer",
+                        "defaultValue": 20
+                    },
+                    "keyVaultCheckMinimumRSAKeySize": {
+                        "type": "string",
+                        "defaultValue": "Disabled"
+                    },
+                    "keyVaultMinimumRSAKeySizeValue": {
+                        "type": "integer",
+                        "defaultValue": 1024
```

Comment:
is 1024 correct here, or should it be 2048?

Reply:
I added them as example, i can change  to 2048 no problem

Reply:
Changed as requested in latest commit

Reply:
The default value "1024" is not in the allowedValues for this parameter, so this will fail the deployment.